src/globals/scss/helpers/_spacing.scss

```diff
@@ -0,0 +1,42 @@
+// Get single spacing unit. We use "tablet" values from settings/spacing as they
+// cascade consistently. However, there is a naming disconnect here as it feels
+// odd to fetch "tablet" values specifically. We should discuss naming in the
+// spacing maps.
+// Arguments:
+// $negative == true returns a negative unit.
+@function govuk-spacing-unit($spacing-map, $negative: false) {
+  $unit: map-get($spacing-map, "tablet");
+  @if ($negative == true) {
+    $unit: -$unit;
+  }
+  @return $unit;
+}
+
+
+// Get responsive spacing maps as set in settings/spacing. These maps should be
+// used wherever possible to standardise responsive spacing.
+// Arguments:
+// $type = margin | padding
+// $property = top \ right \ bottom \ left \ all
+@mixin govuk-spacing-responsive($spacing-map, $type, $property: "all", $important: false) {
```

Comment:
This is a really nice solution, but I wonder if it'd be 'friendlier' for contributors to have two mixins `govuk-margin-responsive` and `govuk-padding-responsive`? I think involving as few arguments as possible would make it easier to remember, but on the downside, we'd need to define two almost identical mixins.

Reply:
Done. I think this was a really good shout because even though it makes the mixin names longer, it somewhat clarifies what they do.